### ISSUE TITLE
[BUG]: Bash@3 3.225.0 breaks handling of quoted args with targetType: filePath

### ISSUE THREAD

**Issue by sphanley** [microsoft/azure-pipelines-tasks#18665]
### Task name

Bash@3

### Task version

3.225.0

### Environment type (Please select at least one enviroment where you face this issue)

- [X] Self-Hosted
- [ ] Microsoft Hosted
- [ ] VMSS Pool
- [ ] Container

### Azure DevOps Server type

dev.azure.com (formerly visualstudio.com)

### Operation system

MacOS

### Description

I'm executing the following (sanitized) task: 

```
 - task: Bash@3
   displayName: 'Run my Script'
   inputs:
     targetType: filePath
     filePath: ./scripts/my-script.sh
     arguments: '--jobposition-inphase "$(System.JobPositionInPhase)" --totaljobs-inphase "$(System.TotalJobsInPhase)"'
```

With the following script: 

```
#!/bin/bash

start=1;
end=$#;
while [ $start -le $end ] 
do
    case $1 in
      -j|--jobposition-inphase)
        jobPositionInPhase=$2
        ;;
      -tj|--totaljobs-inphase)
        totalJobsInPhase=$2
        ;;
    esac
    start=$((start + 1));
    shift 2;
done

echo "jobPositionInPhase: $jobPositionInPhase"
echo "totalJobsInPhase: $totalJobsInPhase"
...
```

In 2.224.0, that produces the following logs, where the parameters are echoed out unquoted:

### Task log

```shell

==============================================================================
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.224.0
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
==============================================================================
Generating script.
Formatted command: exec bash '/Applications/runner/_work/2/s/scripts/my-script.sh' --jobposition-inphase "1" --totaljobs-inphase "1"
========================== Starting Command Output ===========================
/bin/bash /Applications/runner/_work/_temp/d6fc2605-26e9-4885-911b-c6db0f0225aa.sh
jobPositionInPhase: 1
totalJobsInPhase: 1
```

But in 2.225, they're all combined and stored into `BASHV3_INPUT_SCRIPT_ARGS ` which causes the values to come through quoted, which causes errors later in my script when I try to do math:

```
==============================================================================
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.225.0
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
==============================================================================
Generating script.
Formatted command: exec bash '/Applications/runner/_work/7/s/scripts/my-script.sh' $BASHV3_INPUT_SCRIPT_ARGS
========================== Starting Command Output ===========================
/bin/bash /Applications/runner/_work/_temp/1eeef561-5b76-446d-bcb9-77bbd8dc95c6.sh
jobPositionInPhase: "1"
totalJobsInPhase: "1"
```

### Relevant log output
Here's the error I'm getting when I try to evaluate `jobPos=$((jobPositionInPhase - 1))` using my stored variable, which worked in 2.224.0:
```shell
/Applications/runner/_work/7/s/scripts/my-script.sh: line 102: "1": syntax error: operand expected (error token is ""1"")
```

### Aditional info

In this specific example case, I suspect I could fix the issue by removing the quotes around my arguments, like changing `--jobposition-inphase "$(System.JobPositionInPhase)"` to `--jobposition-inphase $(System.JobPositionInPhase)`. But in some cases, I need to pass a value which may be empty, so I need to be able to wrap the expression in quotes, like I was able to on 2.224.0.

**Comment by JSXRED** [comment]
I can confirm that we are facing the same problem

**Comment by aleksandrlevochkin** [comment]
Thank you for reporting this issue. The fix for this has been rolled out. Can you confirm if it is working for you now?

**Comment by sphanley** [comment]
This appears to be resolved, at least in my test case. Thanks for the swift action!

**Comment by JSXRED** [comment]
I can confirm too, in my case 3.225.1 make it working again.

**Comment by merlynomsft** [comment]
Thank you all for your report and confirmation of the fix in 3.225.1.  We apologize for the inconvenience caused by this bug.  Please let us know if you have any additional feedback or questions.
